cmd/bundle/generate/utils_test.go

```diff
@@ -0,0 +1,72 @@
+package generate
+
+import (
+	"bytes"
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config"
+	mocks_client "github.com/databricks/cli/internal/mocks/sdk/client"
+	mock_workspace "github.com/databricks/cli/internal/mocks/sdk/service/workspace"
+	"github.com/databricks/databricks-sdk-go/client"
+	databrickscfg "github.com/databricks/databricks-sdk-go/config"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	_ "go.uber.org/mock/mockgen/model"
+)
+
+func TestDownloadNotebookAndReplaceTaskPath(t *testing.T) {
+	b := &bundle.Bundle{
+		Config: config.Root{
+			Workspace: config.Workspace{
+				RootPath: "/Users/foo@bar.com",
+			},
+		},
+	}
+
+	task := &jobs.Task{
+		NotebookTask: &jobs.NotebookTask{
+			NotebookPath: "/Shared/Notebook/notebook",
+		},
+	}
+
+	w := b.WorkspaceClient()
+	_, err := client.New(&databrickscfg.Config{})
```

Comment:
it can already be done simpler
```suggestion
	_, err := client.New(&databrickscfg.Config{
	   HTTPTransport: fixtures.MappingTransport{
	     "GET /api/2.0/workspace/export?direct_download=true&path=%2Ffoo%2Fbar": {
		    PassFile: "testdata/notebook_with_print",
		 },
	   }
	})
```